### PR TITLE
x11-misc/picom: version bump to 8

### PR DESCRIPTION
**Pull request by AnAverageHuman**
- Remove backported `PATCHES`.
- Upstream removed "compton-convgen.py", so remove from `src_install`.

### CODE REVIEW

File: x11-misc/picom/picom-8.ebuild [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit meson python-r1 xdg
```
Comment (voyageur) [inline x11-misc/picom/picom-8.ebuild:7]: With the conversion script removed, no need anymore for PYTHON_COMPAT and python-r1 eclass

File: x11-misc/picom/picom-8.ebuild [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+DESCRIPTION="A lightweight compositor for X11 (previously a compton fork)"
+HOMEPAGE="https://github.com/yshui/picom"
+SRC_URI="https://github.com/yshui/picom/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0 MIT"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="+config-file dbus +doc +drm opengl pcre"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```
Comment (voyageur) [inline x11-misc/picom/picom-8.ebuild:18]: Same

File: x11-misc/picom/picom-8.ebuild [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+RDEPEND="${PYTHON_DEPS}
```
Comment (voyageur) [inline x11-misc/picom/picom-8.ebuild:20]: And same, no more need for PYTHON_DEPS

File: x11-misc/picom/picom-8.ebuild [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+BDEPEND="virtual/pkgconfig
+	doc? ( app-text/asciidoc )"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use config-file config_file)
+		$(meson_use dbus)
+		$(meson_use doc build_docs)
```
Comment (voyageur) [inline x11-misc/picom/picom-8.ebuild:47]: This option was changed to with_docs https://github.com/yshui/picom/commit/3f2a6718dfb84110862ef3b51b2c961898f21bfd#diff-f2ff94c68cbdd606743b339aa68a4164

File: x11-misc/picom/picom-8.ebuild (hunk 1 above)
Comment (AnAverageHuman) [reply x11-misc/picom/picom-8.ebuild]: Changed to python-any-r1 in an attempt to get tests working. Unfortunately some tests need python-dbus-next, which isn't packaged, so I `RESTRICT=test`.

**Pull request closed without merging**